controllers/servicediscovery/servicediscovery_controller.go

```diff
@@ -36,8 +36,8 @@ import (
 	"github.com/submariner-io/admiral/pkg/util"
 	submarinerv1alpha1 "github.com/submariner-io/submariner-operator/api/v1alpha1"
 	"github.com/submariner-io/submariner-operator/controllers/constants"
-	"github.com/submariner-io/submariner-operator/controllers/helpers"
 	"github.com/submariner-io/submariner-operator/controllers/metrics"
+	reconcile2 "github.com/submariner-io/submariner-operator/controllers/reconcile"
```

Comment:
I can’t think of a better package name right now but the conflict with `sigs.k8s.io/controller-runtime/pkg/reconcile` is unfortunate.

Reply:
Oh I didn't notice that. I can alias this import.

Reply:
OK I renamed `reconcile` to `apply`, so eg `apply.Deployment`.